src/traces/parcoords/parcoords.js

```diff
@@ -145,7 +145,7 @@ function model(layout, d, i, a) {
         unitToColor: unitToColorScale(d.line.colorscale, d.line.cmin, d.line.cmax, d.line.color),
         lines: lines,
         translateX: (d.domain.x[0] || 0) * layout.width,
-        translateY: (d.domain.y[0] || 0) * layout.height,
+        translateY: layout.height - (d.domain.y[1] || 1) * layout.height,
```

Comment:
No need for that `|| 1`, `d.domain.y` will always (unless it's not referencing `fullLayout` values) be filled in here.